target/linux/ar71xx/files/arch/mips/ath79/Kconfig.openwrt

```diff
@@ -1079,7 +1079,15 @@ config ATH79_MACH_WNR2000_V3
 	select ATH79_DEV_LEDS_GPIO
 	select ATH79_DEV_M25P80
 
-	config ATH79_MACH_WNR2200
+config ATH79_MACH_WNR2000_V5
+	bool "NETGEAR WNR2000 V5 board support (test kconfig.openwrt)"
+	select SOC_QCA953X
+	select ATH79_DEV_ETH
+	select ATH79_DEV_GPIO_BUTTONS
+	select ATH79_DEV_LEDS_GPIO
+	select ATH79_DEV_M25P80
+
+config ATH79_MACH_WNR2200
```

Comment:
I think you should add this after _V4, not between V2,3 and original. (even if v3 appears to be in teh wrong place)